domains/sewt.json

```diff
@@ -11,7 +11,10 @@
               "mx1.improvmx.com",
               "mx2.improvmx.com"
             ],
-            "TXT": "v=spf1 include:spf.improvmx.com ~all",
+            "TXT": [
+              "v=spf1 include:spf.improvmx.com ~all",
+              "google-site-verification=bhWE1G1gKHlyzgxlFjPq4o3WMLaXUu4CDgsv3URVh3w"
```

Comment:
we don't allow google site verification as TXT, use the HTTP method instead

Reply:
okay